packages/@react-aria/select/src/HiddenSelect.tsx

```diff
@@ -64,6 +65,7 @@ export function HiddenSelect<T>(props: HiddenSelectProps<T>) {
           <select
             tabIndex={-1}
             name={name}
+            size={state.collection.size}
```

Comment:
where did this change come from?

Reply:
please see PR description:
html select will have role combobox if it has no size, if it does have a size attribute, then it will have role listbox